sormas-api/src/main/java/de/symeda/sormas/api/EntityDtoAccessHelper.java

```diff
@@ -15,12 +30,13 @@
 
 public class EntityDtoAccessHelper {
 
-	public static Object getPropertyValue(Object entity, String propertyKey) throws InvocationTargetException, IllegalAccessException {
+	public static Object getPropertyValue(HasUuid entity, String propertyKey) throws InvocationTargetException, IllegalAccessException {
 		if (entity == null) {
 			return null;
 		}
 		Class<?> entityClass = entity.getClass();
-		while (entityClass != null) {
+		while (entityClass != null
+			&& (EntityDto.class.isAssignableFrom(entity.getClass()) || ReferenceDto.class.isAssignableFrom(entity.getClass()))) {
```

Comment:
Why is this condition:
1. Not present on current development branch
2. Would needlessly reduce the functionality to only some implementations of `HasUuid`.

Reply:
1. in the current development branch, the superclass recursion is restricted by
```java
entityClass = HasUuid.class.isAssignableFrom(superclass) ? (Class<? extends HasUuid>) superclass : null;
```
which leads subsequently to breaking the `while` loop. This is in practice equivalent to the check for type `EntityDto` or `ReferenceDto` (see 2.)
2. Checking for type `HasUuid` is slightly more general than checking for `EntityDto` or `ReferenceDto`, however, the intended use case is `EntityDto` or `ReferenceDto`. Other implementations of `HasUuid` are either irrelevant (`AbstractDomainObject` is not used in this context) or are used in tests only. Thus the more restrictive check for `EntityDto` or `ReferenceDto` is actually more accurate here.

Reply:
However, allowing for `HasUuid` does not hurt here. Removed obsolete checks, too.